zerver/management/commands/export.py

```diff
@@ -126,3 +128,4 @@ def handle(self, *args, **options):
         os.chdir(os.path.dirname(output_dir))
         subprocess.check_call(["tar", "-czf", tarball_path, os.path.basename(output_dir)])
         print("Tarball written to %s" % (tarball_path,))
+        do_write_stats_file_for_realm_export(output_dir)
```

Comment:
hmm, why did you choose not to write it into the tarball?  Seems like it might be handy to have it there.